amundsen_application/config.py

```diff
@@ -72,6 +72,8 @@ class Config:
     PREVIEW_CLIENT_USERNAME = os.getenv('PREVIEW_CLIENT_USERNAME')  # type: Optional[str]
     PREVIEW_CLIENT_PASSWORD = os.getenv('PREVIEW_CLIENT_PASSWORD')  # type: Optional[str]
     PREVIEW_CLIENT_CERTIFICATE = os.getenv('PREVIEW_CLIENT_CERTIFICATE')  # type: Optional[str]
+    PREVIEW_PROJECTS = []  # type : List[Optional[str]]
```

Comment:
could we have a BigqueryPreviewClientConfig with this new  config (I assume only related to BigqueryClient) etc>

Reply:
i'll refactor it so its just going to be part of the implementation after inheriting the base class.
Instead of having it as config.